<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OData Browser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="Oak::View::OakView" name="OakView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="Oak::View::NodeDataView" name="NodeDataView">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>607</width>
             <height>787</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>splitter</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
  </widget>
  <widget class="Oak::View::ActionToolBar" name="toolBar_ModelActions">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Resources/doc_64.png</normaloff>:/images/Resources/doc_64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionLoadFromArchive">
   <property name="text">
    <string>&amp;Load from Archive</string>
   </property>
   <property name="toolTip">
    <string>Load previous version of the settings file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Resources/save_64.png</normaloff>:/images/Resources/save_64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Resources/save-as-64.png</normaloff>:/images/Resources/save-as-64.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Resources/folder_64.png</normaloff>:/images/Resources/folder_64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Oak::View::NodeDataView</class>
   <extends>QWidget</extends>
   <header location="global">NodeDataView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Oak::View::OakView</class>
   <extends>QTreeWidget</extends>
   <header location="global">OakView.h</header>
  </customwidget>
  <customwidget>
   <class>Oak::View::ActionToolBar</class>
   <extends>QToolBar</extends>
   <header location="global">ActionToolBar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
